src/spaceObjects/shipTemplateBasedObject.cpp

```diff
@@ -5,11 +5,19 @@ REGISTER_SCRIPT_SUBCLASS_NO_CREATE(ShipTemplateBasedObject, SpaceObject)
 {
     /// Set the ship template to be used for this station. Stations use ship-templates to define hull/shields/looks
     REGISTER_SCRIPT_CLASS_FUNCTION(ShipTemplateBasedObject, setTemplate);
-    /// [Depricated]
+    /// [Deprecated]
     REGISTER_SCRIPT_CLASS_FUNCTION(ShipTemplateBasedObject, setShipTemplate);
     /// Set the class name of this object. Normally the class name is copied from the template name (Ex "Cruiser") but you can override it with this function.
     REGISTER_SCRIPT_CLASS_FUNCTION(ShipTemplateBasedObject, setTypeName);
     REGISTER_SCRIPT_CLASS_FUNCTION(ShipTemplateBasedObject, getTypeName);
+    REGISTER_SCRIPT_CLASS_FUNCTION(ShipTemplateBasedObject, getClass);
+    REGISTER_SCRIPT_CLASS_FUNCTION(ShipTemplateBasedObject, getSubClass);
+    /// Sets the physics on this object.
+    /// Enabled, static
+    /// Example: setPhysics(true, true)
+    REGISTER_SCRIPT_CLASS_FUNCTION(ShipTemplateBasedObject, setPhysics);
```

Comment:
I think there are a bunch of thing here now that are overcomplicated. And not actually working. I suggest not having a dynamic `setPhysics` on `shipTemplateBasedObject`. As this will be difficult to correctly send to the clients.